pkgs/tools/graphics/mesa-demos/default.nix

```diff
@@ -25,7 +25,8 @@ stdenv.mkDerivation rec {
     freeglut glew libX11 libXext libGL libGLU mesa wayland
     wayland-protocols
   ] ++ lib.optional (mesa ? osmesa) mesa.osmesa ;
-  nativeBuildInputs = [ meson ninja pkg-config ];
+  nativeBuildInputs = [ meson ninja pkg-config wayland ];
+  depsBuildBuild = [ pkg-config ];
```

Comment:
```suggestion
```

Not necessary. Finding pkg-config was never the issue.

Reply:
You are correct. My fault.